addon/components/vertical-collection/component.js

```diff
@@ -308,48 +318,68 @@ const VerticalCollection = Component.extend({
    * - Can be used to respond to the keyboard input for Find (i.e. ctrl+F/cmd+F) to show all elements, which then
    *    allows the list items to be searchable
    *
-   * @property renderAll
+   * @property renderAll // TODO also an argument, how to represent this?
    * @type Boolean
    * @default false
    */
-  renderAll: false,
+  get renderAll() {
+    return this.args.renderAll ?? false;
+  }
 
   /**
    * The tag name used in DOM elements before and after the rendered list. By default, it is set to
    * 'occluded-content' to avoid any confusion with user's CSS settings. However, it could be
    * overriden to provide custom behavior (for example, in table user wants to set it to 'tr' to
    * comply with table semantics).
+   *
+   * @property occlusionTagName // TODO also an argument, how to represent this?
+   * @type String
+   * @default occluded-content
    */
-  occlusionTagName: 'occluded-content',
+  get occlusionTagName() {
+    return this.args.occlusionTagName ?? 'occluded-content';
+  }
 
-  isEmpty: empty('items'),
-  shouldYieldToInverse: readOnly('isEmpty'),
+  get isEmpty() {
+    return !!(this.args.items?.length);
+  }
 
-  virtualComponents: computed('items.[]', 'renderAll', 'estimateHeight', 'bufferSize', function() {
-    const { _radar } = this;
+  get shouldYieldToInverse() {
+    return this.isEmpty;
+  }
 
-    const items = this.items;
+  @cached
```

Comment:
I'm not convinced this observes the contents of `items.[]` as it did previously. The most compatible fix may simply be to return it to being a CP

Reply:
In the current iteration of this PR, I've returned this to a computed